conf/resources/control-decision.json

```diff
@@ -1,7 +1,7 @@
 {
   "dateTime" : "2020-06-01T15:02:00.000Z",
   "messageType" : "IE060",
-  "message" : "<CC060A><SynIdeMES1>UNOC</SynIdeMES1><SynVerNumMES2>3</SynVerNumMES2><MesSenMES3>token</MesSenMES3><SenIdeCodQuaMES4>toke</SenIdeCodQuaMES4><MesRecMES6>token</MesRecMES6><RecIdeCodQuaMES7>toke</RecIdeCodQuaMES7><DatOfPreMES9>tokent</DatOfPreMES9><TimOfPreMES10>toke</TimOfPreMES10><IntConRefMES11>token</IntConRefMES11><RecRefMES12>token</RecRefMES12><RecRefQuaMES13>to</RecRefQuaMES13><AppRefMES14>token</AppRefMES14><PriMES15>t</PriMES15><AckReqMES16>0</AckReqMES16><ComAgrIdMES17>token</ComAgrIdMES17><TesIndMES18>0</TesIndMES18><MesIdeMES19>token</MesIdeMES19><MesTypMES20>CD024A</MesTypMES20><ComAccRefMES21>token</ComAccRefMES21><MesSeqNumMES22>to</MesSeqNumMES22><FirAndLasTraMES23>F</FirAndLasTraMES23><HEAHEA>  <DocNumHEA5>token</DocNumHEA5>  <DatOfConNotHEA148>tokentok</DatOfConNotHEA148></HEAHEA><TRAPRIPC1>  <NamPC17>token</NamPC17>  <StrAndNumPC122>token</StrAndNumPC122><PosCodPC123>token</PosCodPC123><CitPC124>token</CitPC124><CouPC125>to</CouPC125>    <NADLNGPC>to</NADLNGPC>    <TINPC159>string</TINPC159>    <HITPC126>token</HITPC126></TRAPRIPC1><CUSOFFDEPEPT><RefNumEPT1>tokentok</RefNumEPT1></CUSOFFDEPEPT></CC060A>",
+  "message" : "<CC060A>\n    <SynIdeMES1>UNOC</SynIdeMES1>\n    <SynVerNumMES2>3</SynVerNumMES2>\n    <MesSenMES3>token</MesSenMES3>\n    <SenIdeCodQuaMES4>toke</SenIdeCodQuaMES4>\n    <MesRecMES6>token</MesRecMES6>\n    <RecIdeCodQuaMES7>toke</RecIdeCodQuaMES7>\n    <DatOfPreMES9>tokent</DatOfPreMES9>\n    <TimOfPreMES10>toke</TimOfPreMES10>\n    <IntConRefMES11>token</IntConRefMES11>\n    <RecRefMES12>token</RecRefMES12>\n    <RecRefQuaMES13>to</RecRefQuaMES13>\n    <AppRefMES14>token</AppRefMES14>\n    <PriMES15>t</PriMES15>\n    <AckReqMES16>0</AckReqMES16>\n    <ComAgrIdMES17>token</ComAgrIdMES17>\n    <TesIndMES18>0</TesIndMES18>\n    <MesIdeMES19>token</MesIdeMES19>\n    <MesTypMES20>CD024A</MesTypMES20>\n    <ComAccRefMES21>token</ComAccRefMES21>\n    <MesSeqNumMES22>to</MesSeqNumMES22>\n    <FirAndLasTraMES23>F</FirAndLasTraMES23>\n    <HEAHEA>\n        <DocNumHEA5>19IT02110010007A33</DocNumHEA5>\n        <DatOfConNotHEA148>20190913</DatOfConNotHEA148>\n    </HEAHEA>\n    <TRAPRIPC1>\n        <NamPC17>DHL</NamPC17>\n        <StrAndNumPC122>token</StrAndNumPC122>\n        <PosCodPC123>token</PosCodPC123>\n        <CitPC124>token</CitPC124>\n        <CouPC125>to</CouPC125>\n        <NADLNGPC>to</NADLNGPC>\n        <TINPC159>GB652420267000</TINPC159>\n        <HITPC126>token</HITPC126>\n    </TRAPRIPC1>\n    <CUSOFFDEPEPT>\n        <RefNumEPT1>tokentok</RefNumEPT1>\n    </CUSOFFDEPEPT>\n</CC060A>",
```

Comment:
Will the newline characters have any impact?

Reply:
Hi Eric, testing this manually and we were able to read the XML correctly with no impact.